cmd/server/helper_keys.go

```diff
@@ -18,7 +18,7 @@ func createRS256KeysIfNotExist(c *config.Config, set, kid, use string) {
 	if _, err := ctx.KeyManager.GetKey(set, kid); errors.Cause(err) == pkg.ErrNotFound {
 		logrus.Infof("Key pair for signing %s is missing. Creating new one.", set)
 
-		keys, err := generator.Generate("")
+		keys, err := generator.Generate(set)
```

Comment:
I think that would work as long as we are ok with reserved keywords of `public` and `private` referring to the most recent key of that type in the set. Policy could still be written as normal, but the actual key-id would be `public:12345`. Obviously we would need heavy documentation on the matter